### DISCUSSION TITLE
Can I ignore specific webpack-dev-server errors appearing in browser console? Want to ignore websocket not connecting error.

### DISCUSSION THREAD

**Discussion by Codex101** [webpack/webpack#17108] [Q&A]
I have a websocket connection error using webpack 5, webpack dev server and react refresh plugin. The hot reloading does work, but there's a constant repeat of the below error every few seconds. 

```
Errors

WebSocketClient.js:13 WebSocket connection to 'ws://localhost/ws' failed: Error during WebSocket handshake: Unexpected response code: 404
 [webpack-dev-server]  index.js:493
{
  isTrusted: true
  bubbles: false
  cancelBubble: false
  cancelable: false
  composed: false
  currentTarget: WebSocket {url: 'ws://localhost/ws', readyState: 3, bufferedAmount: 0, onopen: ƒ, onerror: ƒ, …}
  defaultPrevented: false
  eventPhase: 0
  path: []
  returnValue: true
  srcElement: WebSocket {url: 'ws://localhost/ws', readyState: 3, bufferedAmount: 0, onopen: ƒ, onerror: ƒ, …}
  target: WebSocket {url: 'ws://localhost/ws', readyState: 3, bufferedAmount: 0, onopen: ƒ, onerror: ƒ, …}
  timeStamp: 25851.299999999814
  type: "error"
}
```

I see that webpack 5 has a `ignoreWarnings` option, and you can set the webpack-dev-server to only show certain errors, warnings etc in the console log. But that doesn't seem to work for errors. 

But, can I suppress this specific websocket error from the module webpack-dev-server? As I still want to show errors in general, just not this one.

**Comment by alexander-akait** [comment]
You should not ignore such errors, check your `publicPath` option

**Comment by Codex101** [comment]
I'm using [electron-react-boilerplate](https://github.com/electron-react-boilerplate/electron-react-boilerplate/blob/main/.erb/configs/webpack.config.renderer.dev.ts), minus the web stuff there. 

Note that this websocket issue didn't occur in webpack 4, only happened after upgrading to webpack 5, webpack cli, dev-server and [react refresh plugin](https://github.com/pmmmwh/react-refresh-webpack-plugin) to latest. 

I think it's related to react refresh plugin not respecting the websocketUrl values, since I have a non-standard url. http:localhost:port/dist, it gives a connection error since it's trying to connect to ws://localhost/ws. 

The package author said they'd find time to do an update, but who knows when that is. Hence the best solution for now would be straight up ignoring the websocket error message that comes from webpack-dev-server and hot reloading

**Reply by alexander-akait** [reply]
Can you create small reproducible test repo and I will look
